standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+Mobile Application Standards
+============================
+
+Scope
+-----
+
+These standards apply to mobile application in-house development and management.
+
+These standards are intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+As implied, there are a number of ways of delivering mobile apps. These standards will cover which approach is best and under what circumstances, and aspects that are common to all or many of the approaches. As and when more detail is available on specific approaches, such as Flutter app development, or Progressive Web Apps, such approach-specific standards should form part of their own separate, detailed, documented standard, which, ideally, will simply be a reference to a recognised external standard for that specific technology.
+
+Caveats
+-------
+
+The huge increase in the use of mobile devices means that mobile application development is now of critical interest to the DEFRA family, even more so given the amount of field-work being carried out by workers within the DEFRA family.
+
+This is a fast moving area involving multiple platforms and diverse technology stacks: these standards are therefore early and provisional. As such, these standards often require more background explanation and justification than those for other, more mature or more homogenous, technology stacks. Inevitably, therefore, these standards, for the time being, are likely to be more discursive than those for other areas, and, unavoidably, the line between standard and guidance can often, as yet, not be so sharply drawn.
+
+This document should therefore be read in conjunction with the Mobile Application Guidance.
+
+Which Mobile Platforms Must Be Supported
+----------------------------------------
+
+Mobile applications must run on mobile devices. Currently DEFRA has adopted Apple technology as the standard platform for mobile devices. **All mobile applications intended for internal use must therefore run on Apple devices. All mobile applications intended for use beyond the DEFRA family, by any form of "extended enterprise" or the general public, must run on at least Apple and Android devices.**
```

Comment:
There seems to be a significant difference between internal and external apps.

It might be worth having two sets of standards to avoid all the "if...then...else..." discussions.

In this case, the "...at least Apple and Android devices" statement for external apps is highly contextual and cannot be mandated.

We _can_ specify a technical standard for internal apps but I think for external ones it depends entirely on understanding the user base.

Reply:
Suspect "Apple and Android" is OK for the "general public" part of that sentence but agree it may not apply to the "extended enterprise". I'll reword as guidance and move to the guidance doc.

Reply:
I'll bear in mind the "is it worth splitting this into internal and external standards" question as I revise, but I suspect there is more in common than different between the two approaches (which itself could imply three docs: common, internal and external, which may be getting too cumbersome), plus it is likely to be more efficient to try and keep internal and external aligned where possible, and it is also useful to be able to easily compare the two and see why there are different.

Reply:
Reworded the bit about "external" platforms to distinguish between the general public and specific user communities, but left all this in standards rather than guidance as it needs the context of the rest of the paragraph.